gitlab-push/handler.go

```diff
@@ -0,0 +1,221 @@
+package function
+
+import (
+	"bytes"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/alexellis/hmac"
+	"github.com/openfaas/openfaas-cloud/sdk"
+)
+
+const (
+	PrivateRepo  = 00
+	InternalRepo = 10
+	PublicRepo   = 20
+	Source       = "gitlab-push"
+	SCM          = "gitlab"
+)
+
+var audit sdk.Audit
```

Comment:
I'm confused why we have `audit` here and not passed to where it is needed?

Reply:
It is how it's done in `github-push` function and I followed the pattern. I can move it inside if necessary